doc/Type/Cool.pod6

```diff
@@ -1494,6 +1501,119 @@ string|/routine/sprintf#Directives>, to format and print the arguments
 =for code
 "%.8f".printf(now - now ); # OUTPUT: «-0.00004118»
 
+=head2 method Complex
+
+Defined as:
+
+    multi method Complex()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.Complex>|/routine/Complex» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Complex;         # OUTPUT: «1+0i␤»
+    say 2e1.Complex;          # OUTPUT: «20+0i␤»
+    say 1.3.Complex;          # OUTPUT: «1.3+0i␤»
+    say (-4/3).Complex;       # OUTPUT: «-1.3333333333333333+0i␤»
+    say "foo".Complex.^name;  # OUTPUT: «Failure␤»
+    say 1+1i.Complex;         # OUTPUT: «1+1i␤»
+
+=head2 method FatRat
+
+Defined as:
+
+    multi method FatRat()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.FatRat>|/routine/FatRat» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Rat;             # OUTPUT: «1␤»
```

Comment:
Maybe you mean FatRat here?

Reply:
Sure ! I'll correct these pastos ! :sweat_smile:

Reply:
In general, and because of this reason, I don't really like content to be exactly the same but for a small variation, for this reason and others. But it's OK as an initial version.

Reply:
I understand.